vllm/executor/ray_gpu_executor.py

```diff
@@ -344,11 +340,16 @@ class RayGPUExecutorAsync(RayGPUExecutor, DistributedGPUExecutorAsync):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.driver_exec_method = make_async(self.driver_worker.execute_method)
+        self.pp_locks: Optional[List[asyncio.Lock]] = None
 
     async def _driver_execute_model_async(
         self,
         execute_model_req: Optional[ExecuteModelRequest] = None
     ) -> List[SamplerOutput]:
+        if not self.tp_driver_workers:
```

Comment:
This is a nice touch - we can probably add it in the Ray version as well

Reply:
@andoorve this is the Ray version :-)

Reply:
Ah of course!